backend/package.json

```diff
@@ -50,7 +50,9 @@
     "mongoose": "^5.9.13",
     "mongoose-unique-validator": "^2.0.3",
     "node-cache": "^5.1.0",
-    "pino-papertrail": "^2.0.0"
+    "pino-papertrail": "^2.0.0",
+    "tlds": "^1.207.0",
+    "validator": "^13.1.17"
```

Comment:
Mmmm, if two new dependencies were added, we should update the package-lock.json... any reason why was not that committed?

Reply:
Thanks JV, I will commit the package-lock.json